packages/crayons-core/src/components/icon/icon.scss

```diff
@@ -1,21 +1,12 @@
 /**
   @prop --icon-color: Color of the icon
-  @prop --icon-size: Size of the icon
 */
 
-:host {
-  display: inline-block;
-}
-
 .icon {
-  display: block;
-  color: var(--icon-color, $color-elephant-900);
-
-  height: var(--icon-size, 12px);
-  width: var(--icon-size, 12px);
+  display: inline-block;
+  color: var(--icon-color, var(--color-elephant-900));
```

Comment:
color-elephant-900 is a scss variable. Please change it to scss variable

Reply:
closed